handlers/dad.js

```diff
@@ -1,35 +1,28 @@
-const rp = require('request-promise');
+const get = require('request-promise');
 const log4js = require('log4js');
 
-const LOG = log4js.getLogger('dad');
+const log = log4js.getLogger('dad');
+
+const handler = async ({ channel }) => {
+  // An attempt will be made
+  try {
+    const { joke } = await get({ 
+      uri: 'https://icanhazdadjoke.com',
+      json: true,
+    });
+
+    // IT'S FATHER TIME
+    channel.send(joke);
+  } catch (error) {
+    // An attempt was made
+    log.error(error);
+
+    channel.send('Error receiving joke from Dad Central Station');
+  }
+};
 
-/**
- * Retrieving a random dad joke from https://icanhazdadjoke.com/
- * via their public API
- */
 module.exports = {
   name: 'dad',
   trigger: '.dad',
-  handler: dad,
+  handler,
```

Comment:
If you name something the same as the key will it default to that? Just making sure the main will be able to find the handler for this

Reply:
Yeah. It's basically a shorthand that's been around for a while. takes a lot of noise out of key object pairing when you want to do something simple.